### ISSUE TITLE
Reproducible Crash In GTA:SA 1.0 US

### ISSUE THREAD

**Issue by question-to-ask** [thelink2012/modloader#96]
Hi there! I have repeatedly experienced a strange crash that I've been unable to exactly determine the cause of. When I play the game and enter in the "all cars explode" cheat, the game nearly always crashes soon after, especially if I'm flying. 
I realize this is an edge case due to the cheat usage, but it's happened often enough that I'm wondering what exactly it could be. When I check the modloader log, it says it is an issue with d3d9.dll. I only have a few small texture mods that mainly just fix some bugs and UI elements. 
When I disable modloader altogether, the crash does not occur. One of the texture mods I am using does have an altered explosion texture, however the crash does not occur if I cause several dozen explosions naturally. 
I have went through each mod and removed them from the folder individually, but haven't been able to keep the crash from happening unless I disable modloader in the modloader.ini.
Here is my [modloader.log](https://github.com/thelink2012/modloader/files/6874425/modloader.log)
Thank you for your time and I apologize if this isn't the right kind of issue to post about!

**Comment by PRinceBhai** [comment]
Do you have any HandLing.cfg in Mod Loader? If so then try to remove it and then try again with Cheat, ModLoader does something to Car Physics or something idk, but if you make Mod Loader edit HandLing.cfg in any way then the cars will go batshit crazy with RIPAZHA. Try it.

**Comment by question-to-ask** [comment]
> 
> 
> Do you have any HandLing.cfg in Mod Loader? If so then try to remove it and then try again with Cheat, ModLoader does something to Car Physics or something idk, but if you make Mod Loader edit HandLing.cfg in any way then the cars will go batshit crazy with RIPAZHA. Try it.

No, I checked the files and there's none that would affect handling. Thank you for the reply though! :)
